test/cypress/support/index.js

```diff
@@ -13,9 +13,502 @@
 // https://on.cypress.io/configuration
 // ***********************************************************
 
+// ***********************************************
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+
+import 'cypress-file-upload'
 import 'cypress-pipe'
-import './commands.js'
-import './output-logs.js'
```

Comment:
@snowteamer Do you remember why you made this change? (Why these files were deleted and combined into one?)

Reply:
Change reverted